lib/common/html_re.js

```diff
@@ -44,7 +44,7 @@ var close_tag   = /<\/[A-Za-z][A-Za-z0-9]*\s*>/;
 var comment     = /<!--([^-]+|[-][^-]+)*-->/;
```

Comment:
```suggestion
var comment     = /<!--[\s\S]*?-->/;
```

Reply:
👀

Reply:
Alternative impl. applied (copied from `markdown-it` again): https://github.com/jonschlinkert/remarkable/pull/335/commits/0c9d32bd4054936526845bfc6d33191ba605b728